dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/messages/messages.properties

```diff
@@ -150,14 +150,15 @@ MoveProjectTimeout=A timeout occurred while moving the {0} project from {1} to {
 
 ProjectDeployedDialogShell=Manage Deployments
 ProjectDeployedDialogTitle=Project Already Deployed
-ProjectDeployedDialogMessage=The project at location {0} is already deployed and enabled on other connections. It is not recommended to have the same project on multiple connections.
-ProjectDeployedDialogGroupLabel=Choose the behaviour for existing enabled deployments
-ProjectDeployedDialogRemoveLabel=Remove
-ProjectDeployedDialogRemoveTooltip=Remove any existing enabled deployments
-ProjectDeployedDialogDisableLabel=Disable
-ProjectDeployedDialogDisableTooltip=Disable any existing enabled deployments
-ProjectDeployedDialogMaintainLabel=Maintain
-ProjectDeployedDialogMaintainTooltip=Do not make any changes to existing enabled deployments
+ProjectDeployedDialogMessage=The project at location {0} is already deployed and enabled on other connections.
+ProjectDeployedDialogInstructions=The project is already deployed on connections: {0}. It is not recommended to have the same project deployed on multiple connections. Choose whether to remove, disable or maintain the existing deployments of this project. Click Cancel to go back and select a different project.
+ProjectDeployedDialogGroupLabel=Choose the behavior for existing enabled deployments
+ProjectDeployedDialogRemoveLabel=Remove existing enabled deployments
+ProjectDeployedDialogRemoveTooltip=Remove existing enabled deployments before adding the project to the {0} connection
+ProjectDeployedDialogDisableLabel=Disable existing enabled deployments
+ProjectDeployedDialogDisableTooltip=Disable existing enabled deployments before adding the project to the {0} connection
+ProjectDeployedDialogMaintainLabel=Maintain existing enabled deployments
+ProjectDeployedDialogMaintainTooltip=Do not make any changes to existing enabled deployments before adding the project to the {0} connection
```

Comment:
For lines 155 to 161, it'd be good to add periods at the end of the sentences. These lines form complete sentences, and 153 and 154 already have periods. :)

For line 161, how about, "Before you add the project to the {0} connection, do not make any changes to existing enabled deployments."? I'm guessing this message appears in the context of adding the project to the connection, so it might be nice to mention that first.

Reply:
Thanks for the review. I updated it according to your suggestion.

Reply:
Since this is a tooltip for the "Maintain" button I think the initial sentence should relate back to the label (Maintain). So I think it is better starting with "Do not make any changes to existing enabled deployments" as that explains what Maintain means.